modules/combined/tests/check_error/contact_displacements_sm.i

```diff
@@ -0,0 +1,83 @@
+[Mesh]
+  file = cube.e
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+[]
+
+[SolidMechanics]
+  [./solid]
+    disp_x = disp_x
+    disp_y = disp_y
+    disp_z = disp_z
+  [../]
+[]
+
+[BCs]
+  [./2_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 2
+    value = 0.0
+  [../]
+  [./2_y]
+    type = DirichletBC
+    variable = disp_y
+    boundary = 2
+    value = 0.0
+  [../]
+  [./2_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = 2
+    value = 0.0
+  [../]
+[]
+
+[Contact]
+  [./fred]
+    master = 1
+    slave = 2
+    disp_x = disp_x
+    disp_y = disp_y
```

Comment:
This is deprecated syntax now and will fail the tests. Pleas put a `displacements = ` into GlobalParams and remove the `disp_[xyz]` from Contact, Pressure, and Density everywhere.